lineage/figures/cartoons/gemcitabine.svg

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+ "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
If these are programmatically generated, they shouldn't be included in the repo.

Reply:
Thanks so much.
Yes, they are generated. Is there a way to remove them after they are called in `genFigures`?

Reply:
Look for how you can delete a file using Python.